src/sentry/models/projectteam.py

```diff
@@ -21,9 +21,11 @@ def get_for_teams_with_org_cache(self, teams: Sequence["Team"]) -> QuerySet["Pro
             .order_by("project__name", "project__slug")
             .select_related("project")
         )
-
         # TODO(dcramer): we should query in bulk for ones we're missing here
```

Comment:
Can remove this comment after we bulk fetch

Reply:
how feels to delete a `# TODO(dcramer):` 
![image](https://github.com/user-attachments/assets/c190c4b0-0a54-4ff6-aaaf-d37a58c77bfe)